client/src/components/TrailShowPage.js

```diff
@@ -23,11 +29,42 @@ const TrailShowPage = (props) => {
 
   useEffect(() => {
     getTrail()
-  }, [])
+  }, [newReview])
```

Comment:
We shouldn't need to check for `newReview` to update, we should set state for the `trail` `reviews` when we receive the new Review

Reply:
done - updating trail state here